src/apps/loyalty/server/template.tsx

```diff
@@ -3,13 +3,20 @@ interface TemplateData {
   html: string
   entrypoint: string
   bootstrapData?: string
-  sharify?: string
+  sharify?: any
   baseURL: string
 }
 
 export default ({styles, html, entrypoint, bootstrapData, sharify, baseURL}: TemplateData) => {
   const fontsURL = "//fast.fonts.net/cssapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.css"
-
```

Comment:
By the way, one main advantage of using the UMD version of `analytics.js` (https://github.com/segmentio/analytics.js) is we can also track events on the server-side.
So for instance

```javascript
import * as analytics from "analytics.js"

export function Inquiries(req: Request, res: Response, next: NextFunction) {
  const { loginPagePath } = artsyPassport.options
  if (!req.user) {
    // Analytics here
    analytics.track("User redirected to login page")
    return res.redirect(req.baseUrl + loginPagePath)
  }

  ...
}
```
Something to consider if we want to track events that occur only on the server-side for instance.